app/controllers/api/v2/groups_controller.rb

```diff
@@ -0,0 +1,83 @@
+class Api::V2::GroupsController < ApplicationController
+  include ApiHelper
+
+
+  def index
+    # TODO: ADD PERMISSIONS
+
+    res = Group.find_by_sql "select * from groups"
```

Comment:
I suggest adding a "TODO" the way I did with all the other queries. I contend this is not a problem for v2 (because we are obsoleting it). We should do it in v3, though.